Jellyfin.Server/Program.cs

```diff
@@ -162,6 +163,24 @@ private static async Task StartServer(IServerApplicationPaths appPaths, StartupO
                     .ConfigureServices(services => appHost.Init(services))
                     .ConfigureWebHostDefaults(webHostBuilder =>
                     {
+                        // Configure Kestrel to listen on the dedicated metrics port if specified
+                        var serverConfig = appHost.ConfigurationManager.Configuration;
+                        if (serverConfig.EnableMetrics && serverConfig.MetricsListenPort > 0)
```

Comment:
This would expose the whole Jellyfin instance on the metrics port, no?

Reply:
Yes, configuring a separate `MetricsListenerPort` would result in the entire backend listening on both ports, with the additional `/metrics` endpoint available on the new additional port. For what it's worth, I was able to connect the server on the metrics port from the web client and configure the server via the web ui. I'm not sure what the downsides of this are in practice, and I didn't test thoroughly to notice a difference.

I don't plan on exposing the metrics port beyond an internal prometheus instance, so I'm not concerned with the implicit behavior of jellyfin being exposed on an additional non-standard port as well.